Requirements.txt

```diff
@@ -0,0 +1,32 @@
+Functionele eisen:
```

Comment:
De fucntionele eisen zijn uitgebreid en goed verdeeld tussen functioneel en niet fucntioneel. Wat goed is om de eisen te nummeren, dat maakt discussie later makkelijker.